javascript/entities/vehicle.js

```diff
@@ -14,6 +14,8 @@ class Vehicle {
     static kVehicleKeysFlip = 8;
     static kVehicleKeysJump = 16;
     static kVehicleKeysNos = 32;
+    static kVehicleKeysBlinkerRight = 64;
+    static kVehicleKeysBlinkerLeft = 256;
```

Comment:
What's wrong with 128? 😛

Reply:
Nothing at all.
I wanted to keep it the same as the keys used in SAMP.

However this doesn't really have any direct couping that way thus set it to 128.